### PR TITLE
feat: set jina log level of gateway to debug

### PR DESCRIPTION
**Pull request by joschkabraun**
Set `JINA_LOG_LEVEL=DEBUG` in gateway in order to get logs there.

### CODE REVIEW

File: now/apps/text_to_text_and_image/flow.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 jtype: Flow
+with:
+  name: nowapi
+  monitoring: true
```
Comment (makram93) [inline now/apps/text_to_text_and_image/flow.yml:4]: Should we set `port_expose` and `prefetch` params like in other flows?

Comment (joschkabraun) [reply now/apps/text_to_text_and_image/flow.yml:4]: I think prefetch is going to be app individual. But I think `port_expose` is good. Thanks!

File: now/apps/base/flow.yml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 jtype: Flow
 version: '1'
 with:
+  name: nowapi
```
Comment (jupyterjazz) [inline now/apps/base/flow.yml:4]: is this flow used somewhere?

**Review by jupyterjazz — APPROVED** [review]
LGTM!

File: now/apps/base/flow.yml (hunk 2 above)
Comment (makram93) [reply now/apps/base/flow.yml:4]: Though, It is used in the base/app.py inside `set_flow_yaml()` but every JinaApp should override this function so in effect this flow is not used. It just serves as a reference template.

**Review by makram93 — APPROVED** [review]

File: now/apps/base/flow.yml (hunk 2 above)
Comment (joschkabraun) [reply now/apps/base/flow.yml:4]: Yes exactly!

**Pull request merged**